db/seeds.rb

```diff
@@ -0,0 +1,18 @@
+Dependency.destroy_all
+Comment.destroy_all
+
+puts "create 2 dependencies: d1 and d2"
```

Comment:
I would remove this. If we want more info about seeds, we can just read this
file, and when we execute the `db:seed` task, rails logs are printed on
standard output if I recall correctly.